code/modules/mob/living/carbon/human/species_types/skeletons.dm

```diff
@@ -55,6 +55,11 @@
 		return TRUE
 	return ..()
 
+/datum/species/skeleton/get_physical_attributes()
+	return "These humerus folk are effectively inorganic, which allows them to resist pressure, temperature, radiation, asphyxiation and even toxins. \
```

Comment:
```suggestion
	return "These humerus folk lack any fleshy biology, which allows them to resist pressure, temperature, radiation, asphyxiation and even toxins. \
```

"organic" refers to being comprised of hydrocarbons, i'm pretty sure these skelebones still have that going on